indico/modules/rb/client/js/common/linking/reducers.js

```diff
@@ -32,7 +33,10 @@ export default (state = initialState, action) => {
         ownRoomName: action.ownRoomName,
         startDt: action.startDt,
         endDt: action.endDt,
+        showNonOverlapping: action.showNonOverlapping || false,
```

Comment:
Is it ever part of the action? If not, I'd always set this to false here.

Reply:
No, it's not. But why setting it to `false`? Shouldn't it be set to the value it already had before?

Reply:
never happens - linking mode is only set once during page load

Reply:
Done in [3d25e5b](https://github.com/indico/indico/pull/6846/commits/3d25e5be379f42e78be9716c4b907eec82dbc2b3).